### PR TITLE
Allows any sharp item for "incise" surgery step (30% success)

### PR DESCRIPTION
**Pull request by Core0verload**
Also splits some of the lines to make `generic_steps.dm` readable on smaller screens.

:cl: CoreOverload
tweak: Any sharp item can now be used for "incise" surgery step, with 30% success probability.
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
aw yes chainsaw incisions

**Comment by Core0verload** [comment]
No, w_class check would stop me from using fire axe as scalpel.

**Comment by XDTM** [comment]
Which is the dream of every surgeon after all

File: code/modules/surgery/generic_steps.dm [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
 	name = "make incision"
-	implements = list(/obj/item/weapon/scalpel = 100, /obj/item/weapon/melee/energy/sword = 75, /obj/item/weapon/kitchen/knife = 65, /obj/item/weapon/shard = 45)
+	implements = list(/obj/item/weapon/scalpel = 100, /obj/item/weapon/melee/energy/sword = 75, /obj/item/weapon/kitchen/knife = 65,
+		/obj/item/weapon/shard = 45, /obj/item = 30) // 30% success with any sharp item.
 	time = 16
 
 /datum/surgery_step/incise/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
-	user.visible_message("[user] begins to make an incision in [target]'s [parse_zone(target_zone)].", "<span class='notice'>You begin to make an incision in [target]'s [parse_zone(target_zone)]...</span>")
+	user.visible_message("[user] begins to make an incision in [target]'s [parse_zone(target_zone)].",
+		"<span class='notice'>You begin to make an incision in [target]'s [parse_zone(target_zone)]...</span>")
```
Comment (coiax) [inline code/modules/surgery/generic_steps.dm]: blah blah line breaks, everyone dicks on me for line breaks, blah blah, line breaks are equal to NAZI CONTENT :'(

Comment (Core0verload) [inline code/modules/surgery/generic_steps.dm]: Line breaks are better than walls of text that don't fit the screen.

Comment (XDTM) [inline code/modules/surgery/generic_steps.dm]: @coiax it's usually accepted for visible_message, since it's two separate sentences

**Comment by lzimann** [comment]
[Here's how I feel about your line breaks](https://github.com/tgstation/tgstation/blob/master/CONTRIBUTING.md#banned-content)

**Comment by KorPhaeron** [comment]
Looks suspiciously like a feature

File: code/modules/surgery/generic_steps.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/surgery/generic_steps.dm]: The rule was made for your backslashed strings @coiax

**Comment by Core0verload** [comment]
Being unable to operate with gar glasses is totally a bug, I'd call it a fix or a tweak.

File: code/modules/surgery/generic_steps.dm [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
 
+/datum/surgery_step/incise/tool_check(mob/user, obj/item/tool)
+	if(implement_type == /obj/item && !tool.is_sharp())
+		return 0
 
+	return 1
```
Comment (optimumtact) [inline code/modules/surgery/generic_steps.dm]: please use the true/false defines

**Comment by MrStonedOne** [comment]
Feature.

**Comment by KorPhaeron** [comment]
This is a good change please reopen it soon

**Pull request merged**